src/kubernetes_cluster/spec/cluster.rs

```diff
@@ -250,6 +252,50 @@ pub open spec fn disable_crash<K: ResourceView, T>() -> Action<State<K, T>, (),
     }
 }
 
+pub open spec fn make_kubernetes_api_busy<K: ResourceView, T>() -> Action<State<K, T>, Option<Message>, ()> {
```

Comment:
Can you rename it to `kubernetes_api_is_busy`?

Reply:
Okay. I hesitated a lot on that and `kubernetes_api_is_busy` doesn't sound like an action.

Reply:
That makes sense. How about `busy_kubernetes_api_rejects_request`

Reply:
`busy_kubernetes_api_rejects_request` sounds better but a bit long.